NRLMMD-GEOIPS/geoips_clavrx
# Call reusable brassy workflow from NRLMDD-GEOIPS/.github

* [x] NO REQUIRED ***existing tests*** (explain why not required)
* [x] Required ***updates to other repos*** complete (see https://github.com/NRLMMD-GEOIPS/.github/pull/20)

# Testing Instructions
Push a tagged commit to this branch in the style of vX.Y.Z and watch the CI go! Check for a new PR and look at its commits. Close the PR and delete the associated branch + tag.

# Summary

This adds a workflow that calls a reusable workflow from https://github.com/NRLMMD-GEOIPS/.github to build release notes using [brassy](https://github.com/biosafetylvl5/brassy).

**DO NOT MERGE THIS BEFORE MERGING THE CALLED WORKFLOW**
